plugins/system/service/service.go

```diff
@@ -30,7 +30,7 @@ func (s *Service) ExtrinsicFailed(spec int, event *storage.Event, paramEvent []s
 
 	for _, param := range paramEvent {
 
-		if param.Type == "DispatchError" {
+		if param.Type == "sp_runtime:DispatchError" {
```

Comment:
It is not reliable. At present, it is based on the corresponding type of ``metadata``. Here, it is more reliable to obtain ``DispatchError`` directly according to the ``index`` of ``paramEvent``.